features/profile/view/References.test.tsx

```diff
@@ -199,22 +200,29 @@ describe("References", () => {
       });
     });
 
-    // Since there aren't references from hosts in the fixture data
-    it("shows no references from hosts", async () => {
+    it("only shows references from hosts", async () => {
```

Comment:
Since the "friend" and "guest" reference type were wrongly swapped, it got me thinking: "what's to stop a future mistake where host references might not show up correctly? Better add a test!"

Looking back, it was strange why I/we didn't just update the fixtures to include a reference that can allow displaying references given by host to be tested (damn our naive past selves 🙈) so this has been done now to hopefully prevent more future regression.

Reply:
hahaha damn our naive past selves 😆